build.sbt

```diff
@@ -16,5 +16,6 @@ parallelExecution in Test := false
 
 libraryDependencies ++= Seq(
   "org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0",
-  "com.novocode" % "junit-interface" % "0.11" % "test"
+  "com.novocode" % "junit-interface" % "0.11" % "test",
+  "com.storm-enroute" %% "scalameter" % "0.8.2"
```

Comment:
Yes, I actually started with JMH but didn’t find a way to make it correctly run the tests. The problem is that JMH has to guess how to *build* your test suite, and the only way you can tell it how to do so is via annotations with very poor composition capabilities. As a result, I was unable to define a configuration generating test data based on the test `@Params` (maybe that’s possible, but I couldn’t figure out how to do so…).
With scalameter, in contrast, the control is not inverted: you can build your test suite as you want and then ask scalameter to benchmark it.

Are there known issues in using scalameter?